x-pack/packages/kbn-elastic-assistant/impl/assistant/settings/evaluation_settings/use_perform_evaluation.tsx

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useMutation } from '@tanstack/react-query';
+import type { HttpSetup, IHttpFetchError, ResponseErrorBody } from '@kbn/core-http-browser';
+import type { IToasts } from '@kbn/core-notifications-browser';
+import { i18n } from '@kbn/i18n';
+import { postEvaluation } from '../../api';
+
+const PERFORM_EVALUATION_MUTATION_KEY = ['elastic-assistant', 'post-knowledge-base'];
```

Comment:
consider removing `c-knowledge-base`

Reply:
Resolved in [7a4f3b9](https://github.com/elastic/kibana/pull/167220/commits/7a4f3b92e3c8eb0f32f53709196bcc09bac72803) 👍.